scripts/ci/libraries/_initialization.sh

```diff
@@ -122,32 +118,44 @@ function initialize_common_environment {
     HOST_OS="$(uname -s)"
     export HOST_OS
 
+    # Home directory of the host user
+    HOST_HOME="${HOME}"
+    export HOST_HOME
+
+    # Sources of Airflow on the host.
+    HOST_AIRFLOW_SOURCES="${AIRFLOW_SOURCES}"
+    export HOST_AIRFLOW_SOURCES
 
     # Add the right volume mount for sources, depending which mount strategy is used
     if [[ ${MOUNT_SOURCE_DIR_FOR_STATIC_CHECKS} == "true" ]]; then
         print_info
         print_info "Mount whole airflow source directory for static checks"
         print_info
         EXTRA_DOCKER_FLAGS=( \
-          "-v" "${AIRFLOW_SOURCES}:/opt/airflow" \
-          "--env" "PYTHONDONTWRITEBYTECODE" \
+            "-v" "${AIRFLOW_SOURCES}:/opt/airflow" \
         )
     elif [[ ${MOUNT_HOST_AIRFLOW_VOLUME} == "true" ]]; then
         print_info
         print_info "Mounting necessary host volumes to Docker"
         print_info
 
         read -r -a EXTRA_DOCKER_FLAGS <<< "$(convert_local_mounts_to_docker_params)"
-        EXTRA_DOCKER_FLAGS+=("-v" "${AIRFLOW_SOURCES}/files:/files" )
+        EXTRA_DOCKER_FLAGS+=( \
```

Comment:
Does the backslash need to be there? Because in line 153 you didn't add it.

Reply:
Indeed. Fixed.